src/common/history.h

```diff
@@ -25,6 +25,14 @@
 struct dt_develop_t;
 struct dt_iop_module_t;
 
+typedef enum dt_hash_history_t
+{
+  DT_HH_INITIAL = 1 << 0,
```

Comment:
I think they are all exclusive, right ? So 0, 1, 2, 3 would have been ok too. Here it looks like as we would use bit-wise operators.

Reply:
And BTW, as this is in a spec maybe more meaningful name: DT_HISTORY_HASH_...

Reply:
no, we can call the hash_history_write() asking for writing whatever hashes at once (usually default and current or initial and current), so the setting needs combination.
Currently I use the same enum for as setting and status. So I can set an other enum for the status (like you suggest). Is that you would like ?

Reply:
Fine with me, I missed the possible combination of multiple hash so ignore this part.

Still HH does not read directly, maybe to you but people coming to the code will certainly be more informed with HISTORY_HASH instead of HH (especially in a .h where some values will be used outside of the history file).

Reply:
Understood. I'll do it.

Reply:
done